index.html

```diff
@@ -1,53 +1,88 @@
-<!-- Copyright 2020 Google LLC. All Rights Reserved.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-==============================================================================-->
-
-<head>
-  <!-- Load three.js -->
-<script src="https://cdn.jsdelivr.net/npm/three@0.106.2/build/three.min.js"></script>
-<!-- Load scatter-gl.js -->
-<!-- Use npm package once https://github.com/PAIR-code/scatter-gl/issues/36 is fixed -->
-<script src="https://storage.googleapis.com/learnjs-data/handtrack_staging/scatter-gl.js"></script>
-<style>
-  .detector-canvas-wrapper, .detector-scatter-gl-container {
-    display: inline-block;
-    vertical-align: top;
-  }
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
+    <meta name="generator" content="Jekyll v3.8.5">
 
-  .detector-scatter-gl-container {
-    border: solid 1px black;
-    position: relative;
-  }
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 
-  /* center the canvas within its wrapper */
-  .detector-scatter-gl-container canvas {
-    transform: translate3d(-50%, -50%, 0);
-    left: 50%;
-    top: 50%;
-    position: absolute;
+    <title>👨 Don't Touch Your Face 👩‍</title>
+<style>
+  *{
+  font-family: Futura;
```

Comment:
As a general comment for modifying font, it is a good idea to provide fallback for people do not have that front in their system. Or deliver that font, which is also a can of worms to optimize, as well as license issue.